src/v2/Apps/Auctions/AuctionsApp.tsx

```diff
@@ -71,16 +72,16 @@ const AuctionsApp: React.FC<AuctionsAppProps> = props => {
       </GridColumns>
 
       {user && (
-        <HorizontalPadding>
-          <Box mt={[2, 4]}>
-            <MyBidsFragmentContainer me={viewer.me} />
-          </Box>
+        <HorizontalPadding pb={2} my={[2, 4]}>
+          <Join separator={<Separator my={[2, 4]} />}>
```

Comment:
@The-Beez-Kneez - note this `Join` component, it's pretty handy: https://palette.artsy.net/elements/layout/join